tests/generator/test_routing.py

```diff
@@ -494,6 +494,30 @@ def test_ip_rule_tos(self):
 TypeOfService=250
 '''})
 
+    def test_use_routes(self):
+        """[networkd] Validate config generation when use-routes DHCP override is used"""
+        self.generate('''network:
+  version: 2
+  ethernets:
+    engreen:
+      dhcp4: true
+      dhcp4-overrides:
+        use-routes: false
```

Comment:
The other dhcp-overrides are tested in tests/generator/test_dhcp_overrides.py   Would this fit in there as well (or instead of)?

Reply:
Yeah, it's not that big a deal. We could put it in both, coming up with a nice case for the routing side, and something more generic for general dhcp overrides testing.